TestPlaybooks/playbook-GmailTest.yml

```diff
@@ -443,7 +447,11 @@ tasks:
               field:
                 value:
                   simple: ID
-          - operator: GetFirstObject
+          - operator: atIndex
```

Comment:
any reason you replaced that?
if the context you are looking for is not a list, `atIndex` will fail.
this is why `GetFirstObject` is used here: if value is a list, return the first entry, otherwise, return the original object (since there is only one)

Reply:
I replaced it because GetFirstObject failed.
Namely - when GetFirstObject was being used, an error was received (Error while parsing argument, or something of that nature). When using atIndex the playbook works.
I still need to look into why GetFirstObject failed